django/db/backends/sqlite3/introspection.py

```diff
@@ -192,9 +192,9 @@ def get_primary_key_column(self, cursor, table_name):
         fields_sql = create_sql[create_sql.index('(') + 1:create_sql.rindex(')')]
         for field_desc in fields_sql.split(','):
             field_desc = field_desc.strip()
-            m = re.match('"?([\w-]+)"?.*PRIMARY KEY.*', field_desc)
+            m = re.match('(?:"(.*)"|([\w-]+)).*PRIMARY KEY.*', field_desc)
```

Comment:
Okay, I think a non-capturing group is appropriate here as we want to match then whole string and use the first or second capture group as necessary.